browser/resources/extensions/brave_overrides/side_bar.ts

```diff
@@ -3,39 +3,36 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
-// @ts-nocheck TODO(petemill): Define types and remove ts-nocheck
+import { injectStyle } from '//resources/brave/lit_overriding.js'
+import { css } from '//resources/lit/v3_0/lit.rollup.js'
+import { ExtensionsSidebarElement } from '../sidebar.js'
 
-import {RegisterStyleOverride} from 'chrome://resources/brave/polymer_overriding.js'
-import {html} from 'chrome://resources/polymer/v3_0/polymer/polymer_bundled.min.js'
+injectStyle(ExtensionsSidebarElement, css`
+  <style>
```

Comment:
Does this work? I think you need to remove the `<style>` tags, as this is actually parsed as CSS not HTML :smile:

(if it is working, I guess the `<style>` is being parsed as some invalid CSS and ignored)

Reply:
Wow, yes it was working with that, oddly enough. Will remove it though, thanks for catching that!

Reply:
lol, I've done it a couple of times :D